torch/csrc/lazy/core/lazy_view.cpp

```diff
@@ -98,6 +101,10 @@ Value ApplyUpdate(Value ir_value, const Alias::UpdateData& update_data) {
       case ViewInfo::Type::kResize:
         result = MakeNode<Resize>(result, view_info.source_shape.sizes().vec());
         break;
+      case ViewInfo::Type::kSqueeze:
+        // TODO: in the next PR, replace this w/ MakeNode<Unsqueeze>
```

Comment:
Unsqueeze? or Squeeze?

Reply:
in the backward pass of update we are **undoing** an operation, so for Squeeze we need to Unsqueeze and vice versa :) 

Maybe I should've added `Unsqueeze` in the same PR. I just like small PRs !!